download_files.py

```diff
@@ -32,6 +32,17 @@ def _download_files(urls_and_outpaths: List[Tuple[str, str]], parent_dir: Path)
         if not outdir.exists():
             os.makedirs(outdir)
         subprocess.run(["wget", url, "-O", full_outpath])
+        # if downloading a zip file, unzip
+        if outpath.suffix == ".zip":
+            # unzip in parent directory
+            subprocess.run(["unzip", "-o", full_outpath, "-d", outdir])
+            # assert that the unzipped directory has same name as .zip file
```

Comment:
why?

Reply:
Huh I was thinking about this more in the context of the script in #7080 to help ensure some consistency in the naming, but now that I think about it this isn't necessary at all. I'll remove!